pkgs/top-level/all-packages.nix

```diff
@@ -889,6 +889,8 @@ with pkgs;
 
   graph-easy = callPackage ../tools/graphics/graph-easy { };
 
+  iat = callPackage ../development/tools/iat { };
```

Comment:
Please find a more appropriate category. We will reorganize this hierarchy at some point but we still try to keep it sorted.

Reply:
Thank you, moved to `pkgs/tools/cd-dvd/`